apps/nrm/models/saved-data.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+/**
+ * Send Data to Data Service
+ *
+ * @param {object} data - data to be sent to data store
+ *
+ * @returns {Promise} -  response from data service
+ */
+const sendDataToBeStored = async(data) => {
+  return new Promise((resolve) => resolve(data));
```

Comment:
@sulthan-ahmed Do we need to stub a mock API request here?
Won't the actual API call to the AWS service be within the modern-slavery-data-service module?

This function will eventually be replaced by a
const dataService = require('modern-slavery-data-service')

const sendDataToBeStored = async(data) => {
return dataService.sendData(data) // the service will return a Promise with the response from AWS
}

Reply:
You answered this on in the sprint planning. cheers @sulthan-ahmed